realm.gypi

```diff
@@ -198,9 +198,14 @@
     {
       "target_name": "realm-sync",
       "type": "none",
-      "dependencies": [ "realm-core" ],
+      "dependencies": [ "realm-core", "OpenSSL" ],
       "link_settings": {
           "libraries": [ "-lrealm-sync<(debug_library_suffix)" ],
+          "conditions": [
+              ["OS=='win'", {
+                "libraries": [ "version.lib" ]
```

Comment:
Why do we need to link against this?

Reply:
There is a call to `GetFileVersionInfoExW` in Realm Sync: https://github.com/realm/realm-sync/blob/c62edffc9411119d3cb45768d33f939a2a44ea5e/src/realm/util/platform_info.cpp#L102